IOS10-HW14-OZMANYAN AMO/Sourses/ViewController.swift

```diff
@@ -0,0 +1,361 @@
+            )
+            
+            let groupSize = NSCollectionLayoutSize(
+                widthDimension: .estimated(192),
+                heightDimension: .absolute(186)
+            )
+            
+            let group = NSCollectionLayoutGroup.vertical(
+                layoutSize: groupSize,
+                subitem: item,
+                count: 1)
+            
+            let section = NSCollectionLayoutSection(group: group)
+            section.interGroupSpacing = .zero
+            section.contentInsets = NSDirectionalEdgeInsets(
+                top: .zero,
+                leading: Metric.layoutIndent,
+                bottom: 60,
+                trailing: Metric.layoutIndent)
+            section.orthogonalScrollingBehavior = .paging
+            
+            section.contentInsets.leading = Metric.layoutIndent
+            section.contentInsets = NSDirectionalEdgeInsets(top: 0, leading:    Metric.layoutIndent, bottom: 50, trailing: 0)
+            
+            return section
+            
+        }
+        
+        private func thirdSection() -> NSCollectionLayoutSection {
+            
+            let itemSize = NSCollectionLayoutSize(
+                widthDimension: .fractionalWidth(1),
+                heightDimension: .absolute(52)
+            )
+            
+            let item = NSCollectionLayoutItem(layoutSize: itemSize)
+            item.contentInsets = NSDirectionalEdgeInsets(
+                top: Metric.topIndentVerticalSection,
+                leading: .zero,
+                bottom: .zero,
+                trailing: .zero)
+            
+            
+            let group = NSCollectionLayoutGroup.vertical(
+                layoutSize: itemSize,
+                subitem: item,
+                count: 1)
+            
+            let section = NSCollectionLayoutSection(group: group)
+            
+            section.contentInsets.leading = Metric.layoutIndent
+            section.contentInsets = NSDirectionalEdgeInsets(top: 0, leading:    Metric.layoutIndent, bottom: 5, trailing: 0)
+            
+            let headerSize = NSCollectionLayoutSize(
+                widthDimension: .fractionalWidth(1.0),
+                heightDimension: .absolute(45))
+            let header = NSCollectionLayoutBoundarySupplementaryItem(
+                layoutSize: headerSize,
+                elementKind: UICollectionView.elementKindSectionHeader,
+                alignment: .top)
+            header.zIndex = Int.max
+            section.boundarySupplementaryItems = [header]
+            
+            let layoutSectionHeaderSize = NSCollectionLayoutSize(widthDimension: .fractionalWidth(0.93),
+                                                                 heightDimension: .estimated(80))
+            let layoutSectionHeader = NSCollectionLayoutBoundarySupplementaryItem(layoutSize: layoutSectionHeaderSize, elementKind: UICollectionView.elementKindSectionHeader, alignment: .top)
```

Comment:
переносим параметры на новые строки - дабы избавится от колбасы кода
```suggestion
            let layoutSectionHeaderSize = NSCollectionLayoutSize(
            widthDimension: .fractionalWidth(0.93),
            heightDimension: .estimated(80))
            let layoutSectionHeader = NSCollectionLayoutBoundarySupplementaryItem(
            layoutSize: layoutSectionHeaderSize,
             elementKind: UICollectionView.elementKindSectionHeader,
              alignment: .top
              )
```